Services/ContentExport.php

```diff
@@ -169,6 +183,16 @@ public function generateTable(ContentType $contentType, $format)
                 }
             }
 
+            foreach ($this->registry->getConverters($contentType) as $converter) {
+                if (isset($columnNames)) {
```

Comment:
@jansanne this logic is used by the creator of the exporter: the first time the columNames are set and things are added to the columNames and after the first loop the columNames are unset